### ISSUE TITLE
Xcode 10 + iOS 12

### ISSUE THREAD

**Issue by southfox** [ManueGE/AlamofireCoreData#3]
Hi there, 

first of all, good repository, we are using AlamofireCoreData for a while since last year, and we are really happy with that. Thanks for sharing this.

I'm having an issue compiling iOS 12 with Xcode 10, the folloging issue appears:

- [ ] Invalid redeclaration of '<-'
```
//
//  Operator.swift
//  Alamofire+CoreData
...
public func <- <T: Insertable>( left: inout T!, right: MapValue?) {
```

Do you want me to send a PR on this? or you guys are working right now to fix the issue?

Thanks again,

Javier.

**Comment by ManueGE** [comment]
Hi @southfox , happy to hear you like this. 

I want to find some time to update the repo to match the latest versions of Swift, Alamofire, and Groot but I have been a little busy lately. 

If you want to send a PR with the change for the moment I'll review it asap and will publish the update. 

Thanks.

**Comment by southfox** [comment]
Hi @ManueGE, thanks for contact me.
I've just pushed a new PR about this pointing to develop, let me know if this is fine for you, I've modified a unit test to check if this is working.
I've tested in my sandbox and looks fine for me.
Thanks,
Javier

**Comment by ManueGE** [comment]
I don't think we actually need that change to check that it still works. What is your point?

Anything else looks ok for me. I am updating the library to the new version of Groot and Alamofire and will release soon.

**Comment by ManueGE** [comment]
Fixed in 2.0.0 

thanks 👍
